src/app/pages/account/users/user-list/user-list.component.html

```diff
@@ -15,93 +15,37 @@
   </ix-page-title-header>
 </ng-template>
 
-<mat-card>
-  <table
-    ix-table
-    matSort
-    matSortDisableClear="true"
-    ixTest="users-table"
-    [ix-table-empty]="!dataSource.filteredData.length"
-    [emptyConfig]="emptyConfigService.defaultEmptyConfig((emptyType$ | async))"
-    [matSortActive]="defaultSort.active"
-    [dataSource]="dataSource"
-    [matSortDirection]="defaultSort.direction"
-    [multiTemplateDataRows]="true"
-    [fixedLayout]="true"
-  >
-    <ng-container matColumnDef="username">
-      <th *matHeaderCellDef mat-header-cell mat-sort-header>
-        {{ 'Username' | translate }}
-      </th>
-      <td *matCellDef="let user; dataSource: dataSource" mat-cell>
-        {{ user.username }}
-      </td>
-    </ng-container>
-
-    <ng-container matColumnDef="uid">
-      <th *matHeaderCellDef mat-header-cell mat-sort-header>
-        {{ 'UID' | translate }}
-      </th>
-      <td *matCellDef="let user; dataSource: dataSource" mat-cell>
-        {{ user.uid }}
-      </td>
-    </ng-container>
-
-    <ng-container matColumnDef="builtin">
-      <th *matHeaderCellDef mat-header-cell mat-sort-header>
-        {{ 'Builtin' | translate }}
-      </th>
-      <td *matCellDef="let user; dataSource: dataSource" mat-cell>
+<div class="table-container">
+  <ix-table2>
+    <thead
+      ix-table-head
+      [columns]="columns"
+      [dataProvider]="dataProvider"
+    ></thead>
+    <tbody
+      ix-table-body
+      [columns]="columns"
+      [dataProvider]="dataProvider"
+      [isLoading]="isLoading$"
+    >
+      <ng-template
+        ix-table-cell
+        [columnIndex]="2"
+        [dataProvider]="dataProvider"
+        let-user
+      >
         {{ user.builtin | yesNo }}
```

Comment:
In the future it would be good to think about how to make this more reusable, for example, by allowing column definitions to accept classes and having `BooleanColumn` class.

Reply:
https://ixsystems.atlassian.net/browse/NAS-122529